cyphers/vigener_cypher.cpp

```diff
@@ -0,0 +1,36 @@
+#include "encryption_methods.hpp"
+
+#include <cstring>
+#include <iostream>
+
+int main()
+{
+    std::string inputString;
+    std::string keyword;
+
+    std::cout << "Enter string to be encrypted: ";
+    std::cin >> inputString;
+    std::cout << "Enter keyword: ";
+    std::cin >> keyword;
+    
+    const uint8_t inputStringLength = inputString.size();
+    char *outputString = new char[inputStringLength];
```

Comment:
Try implementing this without using C-style strings

Reply:
I tried to implement it with `std::string`, but the result was really strange - there was some extra symbols in the output string which mustn't appear there.